azure-pipelines.yml

```diff
@@ -547,12 +549,12 @@ jobs:
       
       echo "== Testing pip install on Python 3.9 =="
       docker run \
-        --rm -v $PYPI_FILES:/mnt/pypi python:3.9 \
+        --rm -v $PYPI_FILES:/mnt/pypi mcr.microsoft.com/mirror/docker/library/python:3.9-slim \
         /bin/bash -c "cd /mnt/pypi && ls && pip install --find-links ./ azure_cli-$CLI_VERSION*whl && az self-test && az --version && sleep 5"
       
       echo "== Testing pip install on Python 3.11 =="
       docker run \
-        --rm -v $PYPI_FILES:/mnt/pypi python:3.11 \
+        --rm -v $PYPI_FILES:/mnt/pypi mcr.microsoft.com/mirror/docker/library/python:3.11-slim \
```

Comment:
It looks like `python:3.11` also exists and there is no need to change to `python:3.11-slim`:

https://mcr.microsoft.com/en-us/product/mirror/docker/library/python/tags

![image](https://github.com/Azure/azure-cli/assets/4003950/aea5ae8e-76a3-4e43-9529-0629278f1a56)

Reply:
`3.9` does not exists. Adding `-slim` to make sure they share the same format.

Reply:
Well, MCR should contain `python:3.9` as it is a popular image.